### ISSUE TITLE
Can't resubscribe

### ISSUE THREAD

**Issue by silviatc** [SachaG/SidebarFeedback#17]
Love the redesign! I'm trying to subscribe again and when I do, I get the following error:

> xxxxxx has unsubscribed, and cannot be resubscribed by you. To prevent false spam complaints, they'll need to resubscribe of their own free will. [212] [subscription-failed] [500]

There isn't a clear path _how_ to resubscribe after this. Any help would be much appreciated.. thanks in advance!

**Comment by SachaG** [comment]
Yeah I think you have to resubscribe in Mailchimp directly. Check out http://sidebar.us2.list-manage1.com/subscribe?u=b5af47765edbd2fc173dbf27a&id=9abeab36bb
